alvr/client_openxr/src/extra_extensions/passthrough_fb.rs

```diff
@@ -10,6 +11,7 @@ pub struct PassthroughFB {
 
 impl PassthroughFB {
     pub fn new(session: &xr::Session<xr::OpenGlEs>) -> xr::Result<Self> {
+        let platform = alvr_system_info::platform();
```

Comment:
Don't do this... that function prints in the logcat the system values like manufacturer and model. To void that, get the platform as a parameter